scripts/test_schemas.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
-docker pull onsdigital/eq-schema-validator
-validator="$(docker run -d -p 5001:5000 onsdigital/eq-schema-validator)"
+docker pull onsdigital/eq-schema-validator:eq-2051-date-validation-mm-yyyy
+validator="$(docker run -d -p 5001:5000 onsdigital/eq-schema-validator:eq-2051-date-validation-mm-yyyy)"
```

Comment:
Will need to be put back before merging. Dependent on [schema validator PR](https://github.com/ONSdigital/eq-schema-validator/pull/51)